src/additionalFunction/getTheLargestId.ts

```diff
@@ -0,0 +1,7 @@
+import { Todo } from '../types/todo';
+
+export function getTheLargestId(list: Todo[]): number {
+  return list
+    .map(todo => todo.id)
+    .sort((a, b) => b - a)[0];
+}
```

Comment:
I like your decision to create new files for these methods.

Reply:
just FYI - u can create a file and name it for example `ToDos.helpers.ts` and put all these functions related to one module (ToDos) there, so you do not need to create a separate file for each function